From 0f03e432a9d60771f50dbeeb83f0138daaa6d018 Mon Sep 17 00:00:00 2001
From: Aaron Kling <webgeek1234@gmail.com>
Date: Fri, 24 Jun 2022 17:17:45 -0500
Subject: [PATCH 3/4] sepolicy: Partial revert Add
 sdcard_posix_contextmount_type attribute

Prebuilt version was wrong.
Fixup was in I0ac259c64a83e3a6608e6b9aa8d1ee446169d501

Change-Id: Ie7451894b32c2696fcdbe620aa352d260c687583
---
 prebuilts/api/31.0/public/attributes | 3 ---
 prebuilts/api/31.0/public/domain.te  | 2 +-
 2 files changed, 1 insertion(+), 4 deletions(-)

diff --git a/prebuilts/api/31.0/public/attributes b/prebuilts/api/31.0/public/attributes
index f4024d7e6..b60c9cc62 100644
--- a/prebuilts/api/31.0/public/attributes
+++ b/prebuilts/api/31.0/public/attributes
@@ -21,9 +21,6 @@ attribute fs_type;
 # All types used for context= mounts.
 attribute contextmount_type;
 
-# All types used for sdcard_posix context= mounts.
-attribute sdcard_posix_contextmount_type;
-
 # All types used for files that can exist on a labeled fs.
 # Do not use for pseudo file types.
 # On change, update CHECK_FC_ASSERT_ATTRS
diff --git a/prebuilts/api/31.0/public/domain.te b/prebuilts/api/31.0/public/domain.te
index 4fa157535..799a2f1c5 100644
--- a/prebuilts/api/31.0/public/domain.te
+++ b/prebuilts/api/31.0/public/domain.te
@@ -509,7 +509,7 @@ neverallow * rootfs:file { create write setattr relabelto append unlink link ren
 
 # Restrict context mounts to specific types marked with
 # the contextmount_type attribute.
-neverallow * {fs_type -contextmount_type -sdcard_posix_contextmount_type}:filesystem relabelto;
+neverallow * {fs_type -contextmount_type}:filesystem relabelto;
 
 # Ensure that context mount types are not writable, to ensure that
 # the write to /system restriction above is not bypassed via context=
-- 
2.30.2

